### DISCUSSION TITLE
Suggest Deletion?

### DISCUSSION THREAD

**Discussion by Lockie85** [rfsbraz/deleterr#63] [Q&A] [answered]
Hey,

I wanted to know if this application provides an option for suggesting items for deletion rather than automatically deleting them? I like the idea of automated deletion, but the additional flexibility of suggested for deletion feels more flexible to me.

Is this possible right now?

Many Thanks

**Comment by Lockie85** [comment]
@rfsbraz could I ask if this is possible? Many thanks.

**Comment by rfsbraz** [comment, marked as answer]
Hi! You could run it using `dry_mode: true` on the configuration file, it will basically tell you what it would have deleted but not do it. Is that what you are looking for?

**Reply by Lockie85** [reply]
Yep this will do the job, thank you.
